infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftCheInstallationLocation.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.openshift.environment;
+
+import javax.inject.Singleton;
+
```

Comment:
javadoc

Reply:
This has been added to @PostConstruct

Reply:
It would be nice to see javadoc on class explaining what is it.

Reply:
Fixed.  Sorry, I missed it.